client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -52,11 +47,11 @@ const DashboardSidebarData = ({ setLogoutAlert }) => {
       icon: <BsFillAwardFill />,
       link: '/achievement',
     },
-    // {
-    //   title: 'Help',
-    //   icon: <BsFillQuestionCircleFill />,
-    //   link: '/dashboard/help',
-    // },
+    {
+      title: 'Account',
+      icon: <BsPerson />,
+      link: '/dashboard/profile',
+    },
```

Comment:
Would it be a good idea to remove `Account` from the sidebar in this PR?

Reply:
Account is the new renaming for Team Management. Something people will use to change admin related things like org name, ,profile pic, etc.

Reply:
Oh cause I tot we're moving `logout` and `account` to the top right as a dropdown from username (eventually. i believe cyril or melvin is currently working on it), and only keeping the pages, so I tot we can remove it here since you're changing it. But either is fine for me